backend/app/transcribe1.py

```
@@ -882,10 +911,15 @@ async def process_audio_with_groq(audio_path: str, status_queue: asyncio.Queue,
                 },
                 "timestamp": datetime.now().isoformat()
             }))
-            logger.info(f"Sent transcription_complete message for Groq transcription")
+            logger.info(f"Sent transcription_complete message for cloud transcription")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)